File: JAVA/Program/Cpu.java
public class Cpu extends Komponen {
  private int jumlahCore;
  private float kecepatanGHz;

  public Cpu() {
  }

  public Cpu(int jumlahCore, float kecepatanGHz, String merk, String nama) {
    super(merk, nama);
    this.jumlahCore = jumlahCore;
    this.kecepatanGHz = kecepatanGHz;
  }

  public void setJumlahCore(int jumlahCore) {
    this.jumlahCore = jumlahCore;
  }

  public void setKecepatanGHz(float kecepatanGHz) {
    this.kecepatanGHz = kecepatanGHz;
  }

  public int getJumlahCore() {
    return jumlahCore;
  }

  public float getKecepatanGHz() {
    return kecepatanGHz;
  }
}

File: PYTHON/Program/Cpu.py
from Komponen import Komponen

class Cpu(Komponen):
  def __init__(self, jumlah_core=0, kecepatan_ghz=0.0, merk="", nama=""):
    super().__init__(merk, nama)
    self.jumlah_core = jumlah_core
    self.kecepatan_ghz = kecepatan_ghz

  def set_jumlah_core(self, jumlah_core):
    self.jumlah_core = jumlah_core

  def set_kecepatan_ghz(self, kecepatan_ghz):
    self.kecepatan_ghz = kecepatan_ghz

  def get_jumlah_core(self):
    return self.jumlah_core

  def get_kecepatan_ghz(self):
    return self.kecepatan_ghz
